custom/icds_reports/utils/__init__.py

```diff
@@ -605,6 +606,34 @@ def create_excel_file(excel_data, data_type, file_format):
     return file_hash
 
 
+def link_callback(uri, rel):
```

Comment:
is this still necessary?

Reply:
bump

Reply:
No, it is no longer needed, I have removed it.